guides/common/modules/con_configuring-virt-who-for-provider.adoc

```diff
@@ -0,0 +1,19 @@
+
+ifeval::["{context}" == "kubevirt-virt-who"]
+
+endif::[]
+
```

Comment:
```suggestion
```

Reply:
Whoa -- must have been asleep! Thanks for catching this.